package.json

```diff
@@ -72,6 +72,7 @@
     "vrtest": "cd apps && cd vr-tests && yarn screener"
   },
   "devDependencies": {
+    "@angular-devkit/schematics": "12.0.2",
```

Comment:
there is nx bug as `@nrlw/workspace` doesn't provide this package as it dependency although it's is using it internaly in few places. -> I'll report this to nrwl folks + send PR to fix it